### PR TITLE
alllow timeserieswidget to specify collection id

### PR DESCRIPTION
**Pull request by mzknksh**
+ hack x scales on interaction plugins
- still missing proper marks on non identical x-scales

### CODE REVIEW

File: packages/dashboards/package.json [hunk 1]
```diff
@@ -148,6 +146,12 @@
         "serve-examples": "ng s examples",
         "serve-examples:prod": "npx local-web-server -p 4200 -d sdk/api-docs-ng2/examples --spa index.html",
         "start": "run-p -lr compodoc:serve build-examples:watch",
+        "symlink": "run-s symlink:*",
```
Comment (stepanstava) [inline packages/dashboards/package.json:149]: I guess you need to add it to "assemble" or are you going to change the build config?

File: packages/dashboards/package.json [hunk 2]
```diff
@@ -148,6 +146,12 @@
...
+        "symlink:bits": "npx rimraf node_modules/@nova-ui/bits && npx symlink-dir ../bits/dist/ node_modules/@nova-ui/bits",
+        "symlink:charts": "npx rimraf node_modules/@nova-ui/charts && npx symlink-dir ../charts/dist/ node_modules/@nova-ui/charts",
+        "symunlink": "run-s symlink_undo:*",
```
Comment (stepanstava) [inline packages/dashboards/package.json:152]: no symlink_undo command

File: packages/charts/package.json [hunk 3]
```diff
@@ -103,6 +102,7 @@
     },
     "scripts": {
         "assemble": "run-s prettier-check lint build-lib:locl build-lib:prod compodoc-custom-theme build-examples:prod",
+        "assemble-dev": "yarn run build-lib:prod",
```
Comment (stepanstava) [inline packages/charts/package.json:105]: for "assemble-dev" would be better to run build-lib with non prod config

**Review by stepanstava — COMMENTED** [review]
lgtm

File: packages/charts/package.json (hunk 3 above)
Comment (mzknksh) [reply packages/charts/package.json:105]: the intention is to have the same output as regular assemble without running too many things repeatedly (to be used when working on some feature and testing it's use in some linked project)

Comment (mzknksh) [reply packages/charts/package.json:105]: but as long as dist-copy is part of build-lib:prod I can just remove it and use that one, I wanted to keep the dist-copy only as part of assemble scripts but I guess I will leave it as it was in the end

**Review by stepanstava — APPROVED** [review]

**Pull request merged**